### DISCUSSION TITLE
Is it possible to export sksurv models to PMML format?

### DISCUSSION THREAD

**Discussion by WeijiaZhang24** [sebp/scikit-survival#274] [General]
I tried using sklearn2pmml and jpmml packages, but they do not support models such as survival random forests. Is it possible to export RSF to pmml format? Or is it relatively straightforward to implement such feature?

**Comment by sebp** [comment]
I'm not familiar with sklearn2pmml and how they save/restore sklearn 
models. It would probably be better if you direct your question at the 
maintainers of sklearn2pmml, who are familiar with the implementation 
details.

**Comment by WeijiaZhang24** [comment]
Thanks. I will contact sklearn2pmml to see what's need to be done. I am also not very familiar with pmml unfortunately.
